components/rewards/Transaction.js

```diff
@@ -40,9 +40,9 @@ function Transaction(props) {
             <Caption color={Colors.secondaryText}>
               {`${date.toLocaleDateString('en-US', options)} • ${storeName}`}
             </Caption>
-            <Subhead>{`$5 reward unlocked`}</Subhead>
+            <Subhead>$5 reward unlocked</Subhead>
```

Comment:
Oh wait are you switching these back? Or in a later PR

Reply:
I pulled from `/constants` in #129, but to be clear this pattern is correct as well if no JSX is used. ESLint just wants you to be clear so it enforces template strings as long as a string is being 'concatenated' with one or more variables